book/MovingToTheGPU.html

```diff
@@ -10,52 +10,52 @@
 realistic visuals in real-time applications like video games. GPUs were traditionally used to
-accelerate _scanline rasterization_ but they have since become programmable and capable of running
-a variety of parallel workloads. Notably, dedicated hardware cores to accelerate ray tracing are
-becoming more common in modern GPUs.
+accelerate _scanline rasterization_ but have since become programmable and capable of running
+a variety of parallel workloads. Notably, some modern GPUs are dedicating hardware cores to
+accelerated ray tracing.
 
-GPUs aren't without limitations. Targeting a GPU requires you to program in a way that is different
-from how you would typically approach a CPU program. Taking full advantage of the hardware requires
-very careful tuning based on its architecture and capabilities (which can vary widely across GPU
-vendors and models). Rendering fully path-traced scenes at real-time rates remains elusive even on
-the most high-end GPUs and is an active and vibrant area of research.
+GPUs aren't without limitations. Targeting a GPU requires a different approach than a typical CPU program.
+Taking full advantage of GPU hardware requires careful tuning based on its architecture and integrated
+capabilities (which can vary widely across vendors and models). Rendering fully path-traced scenes at
+real-time rates remains elusive, even on the most high-end GPUs. This is an active and vibrant area of
+Computer Graphics research.
 
 This book is an introduction to GPU programming by building a simple GPU accelerated path tracer.
 We'll focus on building a renderer that can produce high quality and correct images using a fairly
-simple design. It won't be full-featured and its performance will be limited but it will expose you
-to several fundamental GPU programming concepts. The renderer you'll have built by the end can serve
+simple design. It won't be full-featured and its performance will be limited, however it will expose you
+to several fundamental concepts of GPU programming. By the end, the renderer you have built can serve
 as a great starting point for extensions and experiments with more advanced GPU techniques. We will
 avoid most optimizations in favor of simplicity but the renderer will be able to achieve interactive
 frame rates on a decent GPU.[^ch1]
 
-The book follows a similar progression to [_Ray Tracing In One Weekend_][RTIOW]. I cover some of
+This book follows a similar progression to [_Ray Tracing In One Weekend_][RTIOW]. I cover some of
 the same material but I highly recommend completing _RTIOW_ before embarking on building
-the GPU version since doing so will teach you the path tracing algorithm in a much more approachable
+the GPU version. Doing so will teach you the path tracing algorithm in a much more approachable
 way and it will make you appreciate both the advantages and challenges of moving to a GPU-based
 architecture.
```

Comment:
A rather long run on sentence. Favor RTIOW acronym over full title for consistency.

Reply:
Sounds good!